source/clear-linux/tutorials/telemetry-backend/Telemetry-backend.rst

```diff
@@ -0,0 +1,495 @@
+.. _telemtry-backend:
+
+Create a telemetry backend server in Clear Linux
+################################################
+
+|CLOSIA| includes a telemetrics (telemetry + analytics) solution as part of
+the OS that records events of interest and reports them back to the
+development team via the telemetrics daemon, :command:`telemd`.
+
+End users can enable or disable the telemetry client component of |CL| and
+also redirect where records go if they wish to collect records for themselves
+by setting up and using their own telemetry backend server.
+
+A telemetry backend server consists of two Flask applications: an ingestion
+app, :guilabel:`collector`, for records received from telemetrics-client
+probes; and a web app, :guilabel:`telemetryui`, that exposes several views to
+visualize the telemetry data. The :guilabel:`telemetryui` app also provides a
+REST API to perform queries on the data.
+
+The applications run within a web stack, using the :guilabel:`nginx` web
+server, the :guilabel:`uWSGI` application server, and
+:guilabel:`PostgreSQL` as the underlying database server. For a detailed
+description, visit the `Clear Linux telemetry backend server overview`_.
+
+This tutorial will walk you through creating a telemetrics backend server on
+your local |CL| machine using the :command:`deploy.sh` bash shell script that
+is maintained in a GitHub repository at
+https://github.com/clearlinux/telemetrics-backend. Once the backend server is
+up and running we show you how to redirect telemetry records from the system
+you set up to this new server location.
+
+.. note::
+   The telemetrics functionality adheres to `Intel’s privacy policies`_
+   regarding the collection and use of Personally Identifiable Information
+   (PII) and is Open Source. Specifically, no intentionally identifiable
+   information about the user or system owner is collected.
+
+Prerequisites
+=============
```

Comment:
Remember to use asterisks for level two headings and equal signs for level 3.

Reply:
got it, changed to *